exception DivByZero;
def main() {
    try {
        raise DivByZero;
    } except DivByZero {
        print(42);
    }
}
